airgun/widgets.py

```diff
@@ -1118,6 +1118,47 @@ class EditableEntryCheckbox(EditableEntry):
     edit_field = Checkbox(locator=".//input[@type='checkbox']")
 
 
+class EditableEntryMultiCheckbox(EditableEntry):
+    """Should be used in case :class:`EditableEntry` widget represented not by
+    a field, but by a set of checkboxes.
+    """
+    ITEMS_LOCATOR = ".//p"
+    ITEM_LOCATOR = ".//p/text()[normalize-space(.)='{0}']/preceding-sibling::input"
+
+    def __init__(self, parent, locator=None, name=None, logger=None):
+        if locator and name or not locator and not name:
+            raise TypeError('Please specify either locator or name')
+        locator = (
+            locator or
+            ".//dt[normalize-space(.)='{}']"
+            "/following-sibling::dd[1]".format(name)
+        )
+        super(EditableEntryMultiCheckbox, self).__init__(parent, locator=locator,
+                                                         logger=logger)
+        self.edit_button.click()
```

Comment:
Do not like too much the idea of interact with the UI widget field at initialization stage.

Reply:
If I understood correctly, this is what @oshtaier wanted me to do [1]. I don't like it either.
[1] https://github.com/SatelliteQE/airgun/pull/281#discussion_r252328676